Source/makefile

```diff
@@ -30,23 +30,29 @@ TYPE= SHARED
 #	if two source files with the same name (source.c or source.cpp)
 #	are included from different directories.  Also note that spaces
 #	in folder names do not work well with this makefile.
-SRCS= BeWake.cpp	\
-	ConfigView.cpp	\
-	HueSlider.cpp	\
-	Utils.cpp
+SRCS =	\
+	 BeWake.cpp 	\
+	 ConfigView.cpp 	\
+	 HueSlider.cpp	\
+	 Utils.cpp	\
+
 
 #	specify the resource definition files to use
 #	full path or a relative path to the resource file can be used.
-RDEFS= 
+RDEFS =	\
+                 BeWake.rdef	\
```

Comment:
No need to put it in an extra line, just put the single file right after the "RDEF =" above.